.github/workflows/ci.yml

```diff
@@ -0,0 +1,136 @@
+name: tests-and-publish
```

Comment:
Maybe add a comment where the workflow originated from to save the reference and show that we are using Seita's/your work?